src/spyglass/utils/h5pyfile.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+
+# If SPYGLASS_LINDI is set to 1, use the lindi library to open NWB files.
+# Otherwise, use h5py. Note that even in LINDI mode, if the file path does not
+# end with ".lindi.json", then H5pyFile is equivalent to h5py.File. Thus, the
+# difference only matters when opening LINDI files.
+
+USE_LINDI = os.environ.get("SPYGLASS_LINDI", "0") == "1"
```

Comment:
I'd like to move us away from env var in favor of putting everything in `src/spyglass/setting.py`, and grabbing from the `custom` field in `dj_config`

Reply:
Sounds good. Could you give me some hints about the best way to do this?

Reply:
I think I would add it as an item like `test_mode` at [L145](https://github.com/LorenFrankLab/spyglass/blob/master/src/spyglass/settings.py#L145) and then again at [215](https://github.com/LorenFrankLab/spyglass/blob/master/src/spyglass/settings.py#L215). To fetch it from settings as a bare variable, it would also need a line around [516](https://github.com/LorenFrankLab/spyglass/blob/master/src/spyglass/settings.py#L575). For demo, we should also mirror the name in the [example config](https://github.com/LorenFrankLab/spyglass/blob/master/dj_local_conf_example.json).